client/bootstrap

```diff
@@ -945,19 +945,41 @@ class Server(object):
                                          headers=headers,
                                          files=request_files,
                                          timeout=HTTP_TIMEOUT)
+            elif method == 'stream':
+                log('STREAM %s - %s' % (full_url, local_file))
+                # Don't use streaming request in CI environment
+                if os.environ.get('EAPI_TEST') is not None:
+                    log('NOT STREAM FOR CI %s - %s' % (full_url, local_file))
+                    response = requests.get(full_url,
+                                            data=json.dumps(payload),
+                                            headers=headers,
+                                            files=request_files,
+                                            timeout=HTTP_TIMEOUT)
+                else:
+                    if not local_file:
+                        raise ZtpError('Cant STREAM EOS image file without'
+                                       ' file name and path')
+                    with requests.get(full_url, stream=True) as response:
+                        response.raise_for_status()
+                        with open(local_file, 'wb') as f:
+                            for chunk in response.iter_content(chunk_size=8192):
+                                f.write(chunk)
             else:
-                log('Unknown method %s' % method,
-                    error=True)
+                response = None
```

Comment:
I'd move this line out of `try` block - as response has to be instantiated b/c it's getting returned anyway - so, logically it belongs there

Reply:
Done